charts/digibyte/templates/ingress-p2p.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: traefik.containo.us/v1alpha1
+kind: IngressRouteTCP
+metadata:
+  name: node-ingress-p2p
+spec:
+  entryPoints:
+    - p2p
```

Comment:
The entrypoint should be named differently from the dash chart to avoid conflicts when deploying both charts (same for the other ingrress)?